src/relayer/Relayer.ts

```diff
@@ -112,13 +177,14 @@ export class Relayer {
   async checkForUnfilledDepositsAndFill(sendSlowRelays = true): Promise<void> {
     // Fetch all unfilled deposits, order by total earnable fee.
     const { config } = this;
-    const { acrossApiClient, hubPoolClient, profitClient, spokePoolClients, tokenClient, inventoryClient } =
-      this.clients;
+    const { hubPoolClient, profitClient, spokePoolClients, tokenClient, inventoryClient } = this.clients;
 
     // Flush any pre-existing enqueued transactions that might not have been executed.
     this.clients.multiCallerClient.clearTransactionQueue();
```

Comment:
Opportunistic reflow - I wanted to do this previously but it pushed line 180 over 120 chars and forced it to become 7 lines. Now we can keep it all on 1 line.

```suggestion
    const { hubPoolClient, multiCallerClient, profitClient, spokePoolClients, tokenClient, inventoryClient } = this.clients;

    // Flush any pre-existing enqueued transactions that might not have been executed.
    multiCallerClient.clearTransactionQueue();
```